src/content/docs/agents/net-agent/getting-started/8x-to-9x-agent-migration-guide.mdx

```diff
@@ -0,0 +1,418 @@
+---
+title: .NET Agent 8.x to 9.x Migration Guide
+tags:
+  - Agents
+  - NET agent
+  - Migration guides
+translate:
+  - jp
+metaDescription: 'This guide helps users of the .NET agent migrate from 8.x versions to 9.x'
+redirects:
+  - /docs/agents/net-agent/net-agent-api/net-agent-api-guide
```

Comment:
I'm not really sure what the `redirects` for this doc should be so I just left the ones from the doc I copied to get this one started.

Reply:
No redirects needed since this is a new doc. Thanks for checking!